code/game/objects/structures/wall_fountain.dm

```diff
@@ -0,0 +1,44 @@
+// Due to positioning and having a single-direction icon, this one shouldn't be constructable.
```

Comment:
huh. really? could probably restrict it to certain directions or even just have it always face south and be offset onto the wall to the north. could even use the wall-supported checks for it.